packages/eth2.0-params/src/interface.ts

```diff
@@ -20,17 +18,17 @@ export interface IBeaconParams {
   DEPOSIT_CONTRACT_ADDRESS: number;
 
   // Gwei Values
-  MIN_DEPOSIT_AMOUNT: Gwei;
-  MAX_EFFECTIVE_BALANCE: Gwei;
-  EJECTION_BALANCE: Gwei;
-  EFFECTIVE_BALANCE_INCREMENT: Gwei;
+  MIN_DEPOSIT_AMOUNT: BN;
+  MAX_EFFECTIVE_BALANCE: BN;
+  EJECTION_BALANCE: BN;
+  EFFECTIVE_BALANCE_INCREMENT: BN;
 
   // Initial values
-  GENESIS_SLOT: Slot;
-  GENESIS_EPOCH: Epoch;
+  GENESIS_SLOT: number;
+  GENESIS_EPOCH: number;
   BLS_WITHDRAWAL_PREFIX_BYTE: Buffer;
   GENESIS_FORK_VERSION: Buffer;
-  GENESIS_START_SHARD: Shard;
+  GENESIS_START_SHARD: number;
```

Comment:
We should use the type alias's for convenience

Reply:
It's annoying here because we have circular dependency. Not sure how convinient in params are anyways?

Reply:
> It's annoying here because it causes circular dependency. 
Hmm, i see

> Not sure how convinient in params are anyways?
I guess they're just constants... perhaps add comments beside them to identify what they represent?

Reply:
well last three are pretty obvious and balances already have a comment that they are gweis